9/start

```diff
@@ -34,6 +35,21 @@ fi
 echo "Checking setup"
 ./openvas-check-setup --v9
 
+if [ -f /sasl_passwd_template ]; then
+  echo "Configuring postfix"
+
+  set -o nounset
+  set -o errexit
+  set -o pipefail
+
+  envsubst < "/sasl_passwd_template" > "/etc/postfix/sasl_passwd"
```

Comment:
Interesting method. I will likely change this slightly down the road so we can use either envsubst across the board or swap in sed instead. Either way, nice work.